src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -72,9 +86,12 @@
 
     @Test
     public void getStatisticAboutBananaRepeatedMethodCall() {
+        try {
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
-
+        } catch (IOException e) {
+            System.out.println("Can't write data to file");
+        }
```

Comment:
The try-catch block for `IOException` is not necessary in test methods. Letting the exception propagate will cause the test to fail and provide more detailed information about the failure.